include/openthread/channel_manager.h

```diff
@@ -72,6 +72,15 @@ extern "C" {
  */
 otError otChannelManagerRequestChannelChange(otInstance *aInstance, uint8_t aChannel);
 
+/**
+ * This method gets the channel for the last successfully requested channel change.
```

Comment:
nit: "method" -> "function"

suggestion: "successfully requested channel change" -> "successful call to `otChannelManagerRequestChannelChange()`"